schema_base_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+/* TABLES */
+
+CREATE TABLE patients (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(120) NOT NULL,
+  CONSTRAINT fk_patient_id FOREIGN KEY (patient_id) REFERENCES patients(id),
+);
+
+CREATE TABLE invoices (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  total_amount DECIMAL (8, 2),
+  generated_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+  payed_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+  medical_history_id INT,
+  CONSTRAINT fk_medHistory_id FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE invoice_items (
+  id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
+  unit_price DECIMAL (8, 2),
+  quantity INT NOT NULL,
+  total_price DECIMAL(8, 2),
+  invoice_id INT,
+  treatment_id INT,
+  CONSTRAINT fk_invoice_id FOREIGN KEY (invoice_id) REFERENCES invoices(id),
+  CONSTRAINT fk_treatment_id FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
```

Comment:
- [ ] I would advise you to create the `treatments` table before the `invoice_items` table. The `invoice_items` table has a  constraint that references the `treatments` table and when you run this file, that constraint won't be created because the `treatments` table was not created before the `invoice_items` table. Kindly create the `treatments` table before the `invoice_items` table. 😎 🖥️ 

![image](https://github.com/hernandanielzamora/Clinic/assets/77106962/f3bc2ac2-5a00-44d1-943b-3361e8cd83a4)

Reply:
Thanks for the review! The changes were made